packages/embla-carousel/stories/carousel.stories.tsx

```diff
@@ -29,89 +23,77 @@ export default {
 
 const CarouselCardExample: StoryFn<CarouselProps> = (args) => {
   return (
-    <AriaAnnouncerProvider>
```

Comment:
The AriaAnnouncerProvider is inside SaltProvider which is a decorator in Storybook so not needed here